pkg/api/admin/openshiftcluster.go

```diff
@@ -160,6 +160,11 @@ const (
 	VMSizeStandardM128ms VMSize = "Standard_M128ms"
 	VMSizeStandardG5     VMSize = "Standard_G5"
 	VMSizeStandardGS5    VMSize = "Standard_GS5"
+
+	VMSizeStandardL4s  VMSize = "Standard_L4s_v2"
```

Comment:
This size doesn't exists with the command:
```bash
az vm list-sizes -l $LOCATION -o table
```
The size of `Standard_L4s` does exist in westus2 but the docs do not include the 4 core vm in the v2 series listed here: 
https://docs.microsoft.com/en-us/azure/virtual-machines/lsv2-series.  _(Also it is recommended to use the newer vm types from the documentation)_

Are you including the previous generation version?

If you are including the **Standard_L4s** vm type this will need to change to `"Standard_L4s"`

Reply:
Thanks for review. From what I can tell, the blocker is that the current `azure-sdk-for-go` doesn't yet contain all of the SKUs we intend to support (or any v2 SKUs):

https://github.com/Azure/azure-sdk-for-go/blob/main/services/compute/mgmt/2021-07-01/compute/enums.go#L1798-L1805

We have a request open with Microsoft to update it.

Reply:
Also, I see that the L4s are not available, and will also update accordingly.

Reply:
Yeah there is no L4s_v2 and I'm not sure if you want to include the previous generation L4s vm type.  @jewzaam do we know if we want to do that?  
Should we put this on hold until we can get that answered?